commit d299032743e05571ef326c838a5ecf6ef5b3e9c3
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Fri Feb 4 11:09:10 2022 -0800

    x86-64: Fix strcmp-avx2.S
    
    Change "movl %edx, %rdx" to "movl %edx, %edx" in:
    
    commit b77b06e0e296f1a2276c27a67e1d44f2cfa38d45
    Author: Noah Goldstein <goldstein.w.n@gmail.com>
    Date:   Mon Jan 10 15:35:38 2022 -0600
    
        x86: Optimize strcmp-avx2.S
    
    (cherry picked from commit c15efd011cea3d8f0494269eb539583215a1feed)

diff --git a/sysdeps/x86_64/multiarch/strcmp-avx2.S b/sysdeps/x86_64/multiarch/strcmp-avx2.S
index a0d1c65db11028bc..cdded412a70bad10 100644
--- a/sysdeps/x86_64/multiarch/strcmp-avx2.S
+++ b/sysdeps/x86_64/multiarch/strcmp-avx2.S
@@ -106,7 +106,7 @@ ENTRY(STRCMP)
 # ifdef USE_AS_STRNCMP
 #  ifdef __ILP32__
 	/* Clear the upper 32 bits.  */
-	movl	%edx, %rdx
+	movl	%edx, %edx
 #  endif
 	cmp	$1, %RDX_LP
 	/* Signed comparison intentional. We use this branch to also
